pyaqueduct/client/client.py

```diff
@@ -164,22 +164,23 @@ def get_experiments(
         offset: int,
         title: Optional[str] = None,
         tags: Optional[List[str]] = None,
-        start_date: Optional[date] = None,
-        end_date: Optional[date] = None,
+        start_datetime: Optional[datetime] = None,
+        end_datetime: Optional[datetime] = None,
     ) -> ExperimentsInfo:
         """
         Get a list of experiments
 
         Args:
-        - limit (int): Number of experiments to be fetched
-        - offset (offset): Number of experiments to skip
-        - title (str): Perform search on experiments through title
-        - tags (List[str]): Get experiments having these tags
-        - start_date (date): Start date to filter experiments
-        - end_date (date): End date to filter experiments to
+        - limit: Pagination field, number of experiments to be fetched.
+        - offset: Pagination field, number of experiments to skip.
+        - title: Perform search on experiments through their title and alias.
+        - tags: Get experiments that have these tags.
+        - start_date: Start datetime to filter experiments (timezone aware).
+        - end_date: End datetime to filter experiments to (timezone aware).
```

Comment:
Is getting rid of data types intentional? I think docstrings usually benefit from data types.

Reply:
Yes it is intentional. We have been following Google style docstring on deltaflow control for a long time. Types are already covered the type hints, duplication of it is bad. It leads to mismatches.